site/install.md

```diff
@@ -1,8 +1,10 @@
 # Create and manage the operator
 
+An operator is an application-specific controller that extends Kubernetes to create, configure, and manage instances of complex applications. The Oracle WebLogic Server Kubernetes Operator (the "operator") simplifies the management and operation of WebLogic domains and deployments.
+
 ## Overview
 
-The WebLogic Kubernetes Operator uses Helm to create and deploy any necessary resources and then run the operator in a Kubernetes cluster. Helm helps you manage Kubernetes applications. Helm charts help you define and install applications into the Kubernetes cluster. The operator's Helm chart is located in the `kubernetes/charts/weblogic-operator` directory.
+The operator uses Helm to create and deploy any necessary resources and then run the operator in a Kubernetes cluster. Helm helps you manage Kubernetes applications. Helm charts help you define and install applications into the Kubernetes cluster. The operator's Helm chart is located in the `kubernetes/charts/weblogic-operator` directory.
```

Comment:
Since helm wraps the operator runtime, the phrase 'the operator uses Helm' can be a little confusing.  One needs to deduce that 'operator' in this context is the operator project as a whole, and then immediately switch to thinking the 'operator' in  this context is the operator runtime.      How about:

Helm is used to create and deploy and necessary operator resources and to run the operator in a Kubernetes cluster.   Helm is a framework that helps you manage Kubernetes applications, and `helm charts` help you define and install Helm applications into a Kubernetes cluster.